CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## [3.2.0](https://github.com/commercetools/merchant-center-application-kit/compare/v3.1.0...v3.2.0) (2018-11-30)
+
+#### 🐛 Type: Bug
+
+- `application-shell`
+  - [#162](https://github.com/commercetools/merchant-center-application-kit/pull/162) Improve permission names ([@tdeekens](https://github.com/tdeekens))
+
+#### 💥 Type: Breaking Change
+
+- `application-shell`
+  - [#153](https://github.com/commercetools/merchant-center-application-kit/pull/153) Remove `Avatar` from `application-shell` ([@montezume](https://github.com/montezume))
+
+#### 🔮 Type: Chore
+
+- `mc-html-template`, `mc-http-server`
+  - [#159](https://github.com/commercetools/merchant-center-application-kit/pull/159) chore(http-server): update favicon images for different resolutions ([@emmenko](https://github.com/emmenko))
+
+#### 💅 Type: Enhancement
+
+- `application-shell`
+  - [#155](https://github.com/commercetools/merchant-center-application-kit/pull/155) feat(test-utils): wrap rendered component with React.Suspense, include modals container, document renderWithRedux ([@emmenko](https://github.com/emmenko))
+  - [#158](https://github.com/commercetools/merchant-center-application-kit/pull/158) Add new permissions to project query of app-shell ([@tdeekens](https://github.com/tdeekens))
+
+#### 🚀 Type: New Feature
+
+- `l10n`
+  - [#154](https://github.com/commercetools/merchant-center-application-kit/pull/154) Make locale data work with CJS bundle ([@montezume](https://github.com/montezume))
```

Comment:
This should be a bug fix, not a feature

Reply:
Then the wrong label was assigned I suppose. Will change.

Reply:
Sorry, we're arguing over something rather small. I understand that we want proper release notes but for any consumer it doesn't matter if we call this feature or fix. 

Please just @montezume and @emmenko let me know what you want (given feature I will change this again).

Reply:
just leave it as is it's fine